bcipy/gui/params_form.py

```diff
@@ -338,6 +351,15 @@ def __init__(self, parent, title='BCI Parameters',
         self.SetSizer(vbox)
         self.SetupScrolling()
 
+    def get_parameters_from_file(self, file_name: str):
```

Comment:
We have a load json parameters function in `helpers.load`. Can we leverage or modify that for this use case?